Makefile

```diff
@@ -68,7 +68,7 @@ vet:
 
 .PHONY: staticcheck
 staticcheck:
-	@go install honnef.co/go/tools/cmd/staticcheck@v0.4.3
+	@go install honnef.co/go/tools/cmd/staticcheck@v0.5.1
```

Comment:
Using the latest release: https://github.com/dominikh/go-tools/releases/tag/2024.1.1